boot/zephyr/Kconfig

```diff
@@ -766,4 +728,5 @@ config MCUBOOT_BOOTUTIL_LIB_OWN_LOG
 	bool
 	default n
 
+source "$(ZEPHYR_NRF_MODULE_DIR)/modules/Kconfig.mcuboot"
```

Comment:
:+1: for the pronciple.

Question: should this be first so that we have the possibility to overwrite any mcuboot defaults in our custom nrf `Kconfig.mcuboot`.

Reply:
I moved it to the first line now 👍🏻